.gitignore

```diff
@@ -33,3 +33,6 @@ config/secrets.yml.enc
 config/rmt.local.yml
 
 .env
+
+public/repo
```

Comment:
By doing this, the folder `public/repo` will not be on your system with a `git pull`. Is this what you want?

Reply:
Yeah, I wasn't sure. I wanted to clear all the local mirrored files so I ran `rm -rf public/repo`. And then after re-mirroring, git started showing all the files inside as "untracked".
And you're right, we want to create this dir when pulling. I'll revert this.

Reply:
Thanks, I did this because without the folder being there, you start getting errors during mirroring and letting git take care of everything is better than us writing more code.